libs/ui/common/src/lib/chip/chip.tsx

```diff
@@ -22,17 +22,15 @@ export function Chip({
 			)}
 
 			<span className="flex w-full flex-col px-4">{children}</span>
-
-			{onRemove && (
-				<button
-					type="button"
-					data-testid="remove"
```

Comment:
vorher existierte ein attribbut `data-testid` welches jetzt nicht mehr existent ist. wird das irgendwo in tests benötigt?